seqr/views/utils/permissions_utils.py

```diff
@@ -67,7 +67,7 @@ def decorator(view_func):
 superuser_required = _user_has_policies_and_passes_test(lambda user: user.is_superuser)
 
 
-def _has_analyst_access(project):
+def has_analyst_access(project):
```

Comment:
if this function is being used outside the file it needs a clearer name. Maybe `project_has_analyst_access`

Reply:
Can we change the name to something like `is_internal_project`?

Reply:
no because "internal" isn't a word we use anywhere in the seqr code base and `is_analyst_access_project` is grammatically worse than `project_has_analyst_access`